src/System.Security.Cryptography.X509Certificates/tests/PfxTests.cs

```diff
@@ -104,6 +105,59 @@ public static void ExportWithPrivateKey()
             }
         }
 
+        [Fact]
+        public static void ReadECDsaPrivateKey_WindowsPfx()
+        {
+            using (var cert = new X509Certificate2(TestData.ECDsaP256_DigitalSignature_Pfx_Windows, "Test"))
+            using (ECDsa ecdsa = cert.GetECDsaPrivateKey())
+            {
+                Assert.NotNull(ecdsa);
+
+                if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+                {
+                    AssertEccAlgorithm(ecdsa, "ECDSA_P256");
+                }
+            }
+        }
+
+        [Fact]
+        public static void ReadECDsaPrivateKey_OpenSslPfx()
+        {
+            using (var cert = new X509Certificate2(TestData.ECDsaP256_DigitalSignature_Pfx_OpenSsl, "Test"))
+            using (ECDsa ecdsa = cert.GetECDsaPrivateKey())
+            {
+                Assert.NotNull(ecdsa);
+
+                if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
```

Comment:
Good question.  My approach here was ensure that all platforms agree that the certificates contain ECDSA (capable) private keys; and that Windows can do the extra checks for identifying which one is which (should we go through and try to replace all of the PFX content, for example).

Windows reading OpenSSL is just a special case of "can't we all just get along?"